libs/community/langchain_community/llms/__init__.py

```diff
@@ -406,6 +406,17 @@ def _import_azure_openai() -> Type[BaseLLM]:
     return AzureOpenAI
 
 
+def _import_azure_chat_openai() -> Any:
```

Comment:
we shouldn't be adding deprecated classes here

Reply:
Should I directly import from the new path?

Reply:
yup